config/routes.rb

```diff
@@ -3,4 +3,7 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/projects/:id", to: "projects#show"
+  post "/projects/:project_id/contestant_project", to: "contestant_projects#create"
```

Comment:
Great RESTful routing!!! Other RESTful options for this extension are:
"/projects/:project_id/contestants"
"/project_contestants"